arches/app/search/components/base.py

```diff
@@ -54,6 +57,10 @@ def __init__(self, request=None, user=None):
             for search_filter in SearchComponent.objects.all()
         }
         self.search_filters_instances = {}
+        request_object = (
+            self.request.GET if self.request.method == "GET" else self.request.POST
+        )
+        self.search_request = request_object.dict()
```

Comment:
I think that we should rename this to "self.querystring" because essentially that's what it is.  This would reduce confusion with "self.request"